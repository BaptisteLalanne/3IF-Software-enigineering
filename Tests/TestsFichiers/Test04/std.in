1
44
-1
1
2019-12-15
2020-01-15
0