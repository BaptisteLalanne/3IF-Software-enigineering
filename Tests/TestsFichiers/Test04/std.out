Accès aux données...
Données ajoutées avec succés!
6 capteurs non fiables
Bienvenue dans le Menu Principal
--------------MENU-----------------
Choississez parmi les propositions suivantes
1: Calculer Moyenne
0: Quitter
Votre choix:
Veuillez-renseigner les informations suivantes:
Latitude :
Longitude :
Rayon :
Date de début :
Date de fin :
A bientôt !